### ISSUE TITLE
Player error when opening video on Chrome

### ISSUE THREAD

**Issue by romskidd** [ParticleCore/Iridium#833]
I have the same problem as #831 and #832 

Everytime I go on a new video, I have a "An error occurred. Please try again later. (Playback ID: JXl6dLWRcv3F7p2Q) Learn More", this happen everytime you go on another video on youtube (like you are watching a video, and choose to watch an other one directly on the page), it does not happend when you go on youtube directly with a link.

Yes, I do have tried everything on https://github.com/ParticleCore/Iridium/wiki/Report-a-bug please, don't close it for this reason, I tried on two different computers, different profile, with only Iridium and yes, the problem happen, even if there is only Iridium.

If it's already known, can you explain it by not just closing the topic ? If it's because you don't update the version on chrome anymore ? I supposed 0.2.5 on chrome the same as 1.2.5 on Firefox, so it's the last one right?

By the way, it's new, it happened last few days, yes I have an ablocker, uBlock Origin, but it's disabled on youtube because I use premium, I tried to switch the "allow ads on videos" it doesn't change anything.

**Comment by ParticleCore** [comment]
https://github.com/ParticleCore/Iridium/wiki/Report-a-bug

**Comment by ParticleCore** [comment]
> If it's already known, can you explain it by not just closing the topic ?

If people don't bother reading the report a bug instructions I always link to then there's no point in explaining anything.

**Comment by romskidd** [comment]
Oh amazing, finally you explained that there is a topic about it already, look how much effort it took you to say it's a duplicate! Sorry if people don't obviously see it can be a topic as old as two weeks when most people just got this problem a few days ago. So i'm gonna do what you should have done already in all those topics to avoid people keep asking.

**This is a duplicate of #825 , mainly he doesn't update his addon anymore in the stores but he keep it there so people continue to complain of the same problems and if you're on chrome just uninstall it because there is no available working version for you.**

**Comment by ParticleCore** [comment]
> Oh amazing, finally you explained that there is a topic about it already, look how much effort it took you to say it's a duplicate

Much like actually searching, but it appears you didn't bother to do that either before posting for the first time. Again, I link the instructions for report a bug but people still don't read it, so why should I waste my time with someone like that.

> This is a duplicate of #825 , mainly he doesn't update his addon anymore in the stores but he keep it there so people continue to complain of the same problems and if you're on chrome just uninstall it because there is no available working version for you.

Also don't spread false statements, your interpretation skewed by being upset is not accurate and I will not allow this kind of behavior.

**Comment by gorbiWTF** [comment]
I don't know if I should chime in, but I get the very same error message on Youtube in Firefox without ever having installed the "new" Iridium. So it's either Youtube's fault, or from some other extensions.

**Comment by TheSalx1995** [comment]
Now that I look more into this error, I have the feeling YouTube might have changed something regarding ads. Why? Well, I switched to another add-on to ensure it wasn't an exclusive problem with Iridium. And the problem persists, but interestingly:
- With Iridium, the error is only at the very beginning. You open the video and the error is there, not a chance to load anything, the error is the first thing to appear.
- With the other add-on, the error occurs some seconds after the video began.
And since YouTube shows ads either at the beginning or after some seconds, I'm implying it has something to do with that, as I blocked ads with both add-ons.

But I got to admit that I may be wrong, since I need to check if the error still appears when I allow ads while using either add-on. I'll take a look.
